saleor/dashboard/order/views.py

```diff
@@ -1,34 +1,50 @@
 from __future__ import unicode_literals
 
 from django.contrib import messages
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 from django.db import transaction
 from django.shortcuts import get_object_or_404, redirect
 from django.template.context_processors import csrf
 from django.template.response import TemplateResponse
 from django.utils.translation import pgettext_lazy
-from django.views.generic import ListView
 from django_prices.templatetags.prices_i18n import gross
 from prices import Price
 
+from ..order.forms import OrderFilterForm
 from ...order.models import Order, OrderedItem, OrderNote
 from ...userprofile.i18n import AddressForm
-from ..views import (FilterByStatusMixin, StaffMemberOnlyMixin,
-                     staff_member_required)
+from ..views import staff_member_required
 from .forms import (CancelItemsForm, CancelOrderForm, CapturePaymentForm,
                     ChangeQuantityForm, MoveItemsForm, OrderNoteForm,
                     RefundPaymentForm, ReleasePaymentForm, RemoveVoucherForm,
                     ShipGroupForm, CancelGroupForm)
 
 
-class OrderListView(StaffMemberOnlyMixin, FilterByStatusMixin, ListView):
-    template_name = 'dashboard/order/list.html'
-    paginate_by = 20
-    model = Order
-
-    def get_queryset(self):
-        qs = super(OrderListView, self).get_queryset()
-        return qs.prefetch_related(
-            'groups', 'payments', 'groups__items').select_related('user')
+@staff_member_required
+def order_list(request):
+    orders = Order.objects.prefetch_related(
+        'groups', 'payments', 'groups__items').all()
+
+    active_status = request.GET.get('status')
+    if active_status:
+        orders = orders.filter(status=active_status)
+
+    paginator = Paginator(orders, 20)
```

Comment:
We have helper method for that already:
https://github.com/mirumee/saleor/blob/master/saleor/core/utils/__init__.py#L73